test/unittests/commands/xmlupload/test_ontology_problem_models.py

```diff
@@ -1,88 +1,147 @@
 import pandas as pd
 from pandas.testing import assert_frame_equal
 
-from dsp_tools.commands.xmlupload.models.ontology_problem_models import InvalidOntologyElementsInData
+from dsp_tools.commands.xmlupload.models.ontology_lookup_models import TextValueData
+from dsp_tools.commands.xmlupload.models.ontology_problem_models import (
+    InvalidOntologyElementsInData,
+    InvalidTextValueEncodings,
+    _make_msg_for_one_resource,
+    _make_msg_from_df,
+)
 
 
-def test_print_problem_string_cls() -> None:
-    onto = InvalidOntologyElementsInData([("clsA", ["idA"], "wrong")], [], ontos_on_server=["test"])
-    msg = onto._compose_problem_string_for_cls()
-    assert msg == (
-        "The following resource(s) have an invalid resource type:\n\n"
-        "    Resource Type: 'clsA'\n"
-        "    Problem: 'wrong'\n"
-        "    Resource ID(s):\n"
-        "    - idA"
-    )
+class TestInvalidOntologyElementsInData:
+    def test_print_problem_string_cls(self) -> None:
```

Comment:
I just put this into a class nothing else was changed.

Reply:
👍